content/doc/developer/book.html.haml

```diff
@@ -1,9 +1,15 @@
 ---
 title: "Jenkins Developer Reference"
-layout: developerguide
+layout: developer
```

Comment:
<details>

<summary>Before</summary>
![image](https://user-images.githubusercontent.com/21194782/74084249-3cc64300-4a65-11ea-8410-5f0a6d3fbb6a.png)

</details>

<details>

<summary>After</summary>
![image](https://user-images.githubusercontent.com/21194782/74084257-4b145f00-4a65-11ea-99c3-c4040102b7cc.png)
</details>

I would like to just kill this page and replace it with a landing page really, it doesn't need to exist, but I think it looks better than before with this change